packages/story-editor/src/elements/video/playPauseButton.js

```diff
@@ -209,6 +210,33 @@ function PlayPauseButton({
     };
   }, [checkMouseInBBox]);
 
+  const handleKeyDown = useCallback(
+    ({ key }) => {
+      if (allowKeyboardSupport && key === ' ') {
+        const videoNode = getVideoNode();
+        if (videoNode) {
+          if (videoNode.paused) {
+            videoNode.play();
+            setIsPlaying(true);
```

Comment:
[`play()`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/play) returns a Promise that can be rejected if playback fails for whatever reason.

It'd be safer to do something like this:

```js
videoNode
  .play()
  .then(() => setIsPlaying(true))
  .catch(() => {});
```